docs/content/v0.20/examples.md

```diff
@@ -0,0 +1,588 @@
+# KubeStellar Usage Examples
+
```

Comment:
Maybe put X specific files one for each scenario and put here a one liner title + a link ?